### PR TITLE
Vddk container for CNV VMware VM import

### PR DESCRIPTION
**Pull request by apinnick**
Formatted doc preview: http://file.tlv.redhat.com/~apinnick/vddk_container/cnv/cnv_users_guide/cnv-importing-vmware-vm.html

### CODE REVIEW

**Review by mareklibra — CHANGES_REQUESTED** [review]
LGTM in general. Let's just give the user more freedom in choosing repository and responsibility when manipulating with 3rd-party library.

**Review by mareklibra — APPROVED** [review]

**Review by mareklibra — COMMENTED** [review]
lgtm when confirmed that the page is rendered correctly (does not look well on GitHub)

File: modules/cnv-creating-vddk-image.adoc
Comment (mareklibra) [reply modules/cnv-creating-vddk-image.adoc]: Thank you, reading it with context is much better.
It's great to see the disclaimer, it addresses my concerns written above. And the `<1>` renders perfectly there as well.

I can not see any issue in the instructions. So - lgtm.

**Comment by apinnick** [comment]
@kalexand-rh Please review.

**Comment by vikram-redhat** [comment]
@openshift/team-documentation - folks on the roster should review this.

File: modules/cnv-creating-vddk-image.adoc
Comment (jboxman) [inline modules/cnv-creating-vddk-image.adoc]: s/Navigate to/In a web browser, open the/

File: modules/cnv-creating-vddk-image.adoc
Comment (jboxman) [inline modules/cnv-creating-vddk-image.adoc]: s/*My VMware* credentials to download the VDDK SDK/You need your *My VMware* to download the VDDK SDK./

File: modules/cnv-creating-vddk-image.adoc
Comment (jboxman) [inline modules/cnv-creating-vddk-image.adoc]: s/Private container repository./You need a private container repository./

File: modules/cnv-creating-vddk-image.adoc
Comment (jboxman) [inline modules/cnv-creating-vddk-image.adoc]: s/for/for the/

File: modules/cnv-creating-vddk-image.adoc
Comment (jboxman) [inline modules/cnv-creating-vddk-image.adoc]: s/*_Dockerfile_*/`Dockerfile`/

File: modules/cnv-creating-vddk-image.adoc
Comment (jboxman) [inline modules/cnv-creating-vddk-image.adoc]: s/Select the latest VDDK release and/Select the latest VDDK release, and then/

File: modules/cnv-creating-vddk-image.adoc
Comment (jboxman) [inline modules/cnv-creating-vddk-image.adoc]: s/Build the image and/Build the image, and then/

File: modules/cnv-creating-vddk-image.adoc
Comment (jboxman) [inline modules/cnv-creating-vddk-image.adoc]: s/Add the `vddk-init-image`/Add the `vddk-init-image` parameter/

**Comment by jboxman** [comment]
@apinnick, I left some review comments. Per our [contributing guidelines](https://github.com/openshift/openshift-docs/blob/enterprise-4.1/contributing_to_docs/doc_guidelines.adoc#general-file-guidelines), we're wrapping text at 80 columns.

**Comment by jboxman** [comment]
@apinnick which version of OpenShift is this targeted for? Thanks.

File: modules/cnv-creating-vddk-image.adoc
Comment (apinnick) [reply modules/cnv-creating-vddk-image.adoc]: "Open" a web site? Are you sure? I did a global search and this phrasing is not used in Openshift docs.

File: modules/cnv-creating-vddk-image.adoc
Comment (apinnick) [reply modules/cnv-creating-vddk-image.adoc]: Full sentences are unnecessary in a bullet list.

File: modules/cnv-creating-vddk-image.adoc
Comment (apinnick) [reply modules/cnv-creating-vddk-image.adoc]: Same as above.

File: modules/cnv-creating-vddk-image.adoc
Comment (apinnick) [reply modules/cnv-creating-vddk-image.adoc]: I'm willing to add "then" in case some poor user tries to select a release and click a button at the same time but a serial comma is unnecessary in a series of two items.

File: modules/cnv-creating-vddk-image.adoc
Comment (apinnick) [reply modules/cnv-creating-vddk-image.adoc]: > @apinnick which version of OpenShift is this targeted for? Thanks.

CNV 2.1.

**Comment by apinnick** [comment]
@openshift/team-documentation - please review

**Comment by apinnick** [comment]
@vikram-redhat This needs further review or merging.

**Comment by vikram-redhat** [comment]
@openshift/team-documentation PTAL for peer review.

**Comment by mburke5678** [comment]
@apinnick LGTM Is this ready for merge?

**Comment by apinnick** [comment]
@mburke5678 

Yes, it can be merged. Thanks!

**Pull request merged**
